.gitignore

```diff
@@ -1,7 +1,9 @@
-image-syncer
```

Comment:
Maybe configuration of `.gitignore` file should not be included to a commit if it seems personal and piecemeal.

And the `image-syncer` binary file below should not be included in a commit either.